### ISSUE TITLE
Mixcloud failing

### ISSUE THREAD

**Issue by baliw** [rg3/youtube-dl#4834]
```
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-c', '--no-mtime', 'http://www.mixcloud.com/pedro-silva/toper-edu-deep-fog-progged-numix-2-year-anniversary-jan-2015/', '--verbose']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.01.30.2
[debug] Python version 2.7.6 - Darwin-14.1.0-x86_64-i386-64bit
[debug] exe versions: ffmpeg 2.1.3, ffprobe 2.1.3
[debug] Proxy map: {}
[mixcloud] pedro-silva-toper-edu-deep-fog-progged-numix-2-year-anniversary-jan-2015: Downloading webpage
[mixcloud] pedro-silva-toper-edu-deep-fog-progged-numix-2-year-anniversary-jan-2015: Checking URL 1/31 ...
[mixcloud] pedro-silva-toper-edu-deep-fog-progged-numix-2-year-anniversary-jan-2015: Checking URL 2/31 ...
[mixcloud] pedro-silva-toper-edu-deep-fog-progged-numix-2-year-anniversary-jan-2015: Checking URL 3/31 ...
[mixcloud] pedro-silva-toper-edu-deep-fog-progged-numix-2-year-anniversary-jan-2015: Checking URL 4/31 ...
[mixcloud] pedro-silva-toper-edu-deep-fog-progged-numix-2-year-anniversary-jan-2015: Checking URL 5/31 ...
[mixcloud] pedro-silva-toper-edu-deep-fog-progged-numix-2-year-anniversary-jan-2015: Checking URL 6/31 ...
[mixcloud] pedro-silva-toper-edu-deep-fog-progged-numix-2-year-anniversary-jan-2015: Checking URL 7/31 ...
[mixcloud] pedro-silva-toper-edu-deep-fog-progged-numix-2-year-anniversary-jan-2015: Checking URL 8/31 ...
[mixcloud] pedro-silva-toper-edu-deep-fog-progged-numix-2-year-anniversary-jan-2015: Checking URL 9/31 ...
[mixcloud] pedro-silva-toper-edu-deep-fog-progged-numix-2-year-anniversary-jan-2015: Checking URL 10/31 ...
[mixcloud] pedro-silva-toper-edu-deep-fog-progged-numix-2-year-anniversary-jan-2015: Checking URL 11/31 ...
[mixcloud] pedro-silva-toper-edu-deep-fog-progged-numix-2-year-anniversary-jan-2015: Checking URL 12/31 ...
[mixcloud] pedro-silva-toper-edu-deep-fog-progged-numix-2-year-anniversary-jan-2015: Checking URL 13/31 ...
[mixcloud] pedro-silva-toper-edu-deep-fog-progged-numix-2-year-anniversary-jan-2015: Checking URL 14/31 ...
WARNING: unable to extract like count; please report this issue on http://yt-dl.org/bug
WARNING: unable to extract play count; please report this issue on http://yt-dl.org/bug
ERROR: Unable to extract upload date; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/Users/dan/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 618, in extract_info
    ie_result = ie.extract(url)
  File "/Users/dan/bin/youtube-dl/youtube_dl/extractor/common.py", line 266, in extract
    return self._real_extract(url)
  File "/Users/dan/bin/youtube-dl/youtube_dl/extractor/mixcloud.py", line 96, in _real_extract
    webpage, 'upload date'))
  File "/Users/dan/bin/youtube-dl/youtube_dl/extractor/common.py", line 522, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract upload date; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

**Comment by dstftw** [comment]
This is already fixed and will be available in the next version. Thanks for the report.

**Comment by phihag** [comment]
I believe this issue to be fixed in youtube-dl 2015.02.02. See [our FAQ](https://yt-dl.org/update) if you need help updating.
